.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,29 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
Probably don't need to commit this file.

Reply:
Yeah in general we don't need `.idea` files at all.

Reply:
Kept all .idea files the way they were before.